.github/workflows/deploy.yml

```diff
@@ -15,70 +15,85 @@ env:
   AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
 
 jobs:
-  deploy:
+  build_and_push:
```

Comment:
Is this 'build and push to docker hub'? Could it be called that to be clear where the push is to?

Reply:
Done! 👍